Assets/Scripts/WorkPuzzle/FallTimer.cs

```diff
@@ -12,32 +12,52 @@ public class FallTimer : MonoBehaviour
     public WorkManager workManager;
 
     //time between the spawning of each word
-    private float delay = 1f;
-    private float nextTime = 0f;
+    public static float delay = 1f;
+    public static float nextTime = 0f;
 
     //boolean tracks whether or not the user has lost the game 
     public static bool stop;
 
-    private string[] passage;
+    public static string[] passage;
 
     private void Start()
     {
         stop = false;
-        passage = PassageGenerator.GetRandomPassage();
+        passage = PassageGenerator.GetPassage();
     }
 
     private void Update()
-    {
+    {/*
         if (Time.time >= nextTime && !stop)
         {
             workManager.NewWord(passage);
             nextTime = Time.time + delay;
             delay *= .99f;
         }
+        */
+
+        //TO-DO: one passage per session. Instead, level up, then game is complete.
+        //Store the current level so the player can start next level on the next day. 
         if (PassageGenerator.levelUp)
         { //player has completed level, choose another passage to type
-            passage = PassageGenerator.GetRandomPassage();
+            passage = PassageGenerator.GetPassage();
```

Comment:
Move this logic into a public method and have someone call it instead of checking this condition every update.